pkg/ingester/encoding_test.go

```diff
@@ -5,205 +5,15 @@ import (
 	"testing"
 	"time"
 
+	"github.com/grafana/loki/pkg/ingester/wal"
```

Comment:
nit, can we keep all the loki dependencies together at the bottom of the list

Reply:
Sure thing, resolving all